common/config/rush/pnpm-lock.yaml

```diff
@@ -16749,6 +16751,25 @@ packages:
       - supports-color
     dev: false
 
+  file:projects/perf-ai-language-text.tgz:
+    resolution: {integrity: sha512-e+Kh4BFqMg7DolFQ7Rj0jYHLOAoDk4p5NfitePcUIZ+FK01B1QSQjUnroEAorvVwzHimodc3lDEMRhyBsCjzSw==, tarball: file:projects/perf-ai-language-text.tgz}
+    name: '@rush-temp/perf-ai-language-text'
+    version: 0.0.0
```

Comment:
1.0.0?

Reply:
This file is generated by the rush tool and I can't edit it. I have the version for this package set as 1.0.0 in its package.json.